data/json/vehicleparts/frames.json

```diff
@@ -41,6 +41,19 @@
     "location": "structure",
     "durability": 400,
     "difficulty": 1,
+    "requirements": {
+      "install": {
+        "skills": [ [ "mechanics", 1 ] ],
+        "time": 120000,
+        "qualities": [ { "id": "GLARE", "level": 2 } ],
+        "tools": [ [ [ "welder", 50 ], [ "welder_crude", 75 ], [ "toolset", 75 ], [ "oxy_torch", 20 ] ] ],
+        "components": [ [ [ "frame", 1 ] ] ]
+      },
+      "removal": {
+        "skills": [ [ "mechanics", 1 ] ],
+        "qualities": [ { "id": "SAW_M", "level": 1 } ]
```

Comment:
Two balancing changes here:
* Wrench is no longer required. Kinda makes sense because the frame is welded on.
* You can use a multitool or a swiss army knife to remove a frame. This one sounds like a major, unintentional omission.